app/models/account.rb

```diff
@@ -36,6 +36,7 @@
 #  followers_count         :integer          default(0), not null
 #  following_count         :integer          default(0), not null
 #  last_webfingered_at     :datetime
+#  activitystreams2_url    :string
```

Comment:
I'm for `activitystrams2_url` because Webfinger does not tell it has ActivityPub extensions such as `inbox` and `outbox`. Methods referring to this column must NOT assume it has ActivityPub extensions and must do some checks. I'm worrying that naming it as `activitypub_url` would promote such mistakes.